src/components/app/details/cicdHistory/TriggerDetails.tsx

```diff
@@ -87,17 +90,25 @@ export const TriggerDetails = React.memo(
     },
 )
 
-const Finished = React.memo(({ status, finishedOn, artifact }: FinishedType): JSX.Element => {
+const Finished = React.memo(({ status, startedOn, finishedOn, artifact }: FinishedType): JSX.Element => {
```

Comment:
Could you please add Finished prop type? It is missing here

Reply:
I have given the type in FinishedType